lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -291,6 +324,11 @@ testRule({
 			fixed: 'p { width: -moz-fit-content; }',
 			message: messages.rejected('width'),
 		},
+		{
+			code: 'p { width: -moz-fit-content; width: -moz-fit-content !important; }',
+			fixed: 'p { width: -moz-fit-content !important; }',
+			message: messages.rejected('width'),
+		},
```

Comment:
[suggest] To cover the following line,

```
lib/rules/declaration-block-no-duplicate-properties/index.js#L125

Added line #L125 was not covered by tests
```

could you please add a more test case?

```suggestion
		},
		{
			code: 'p { width: 100% !important; width: -moz-fit-content; }',
			fixed: 'p { width: 100% !important; }',
			message: messages.rejected('width'),
		},
```